CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 4.3.0
+
+- Implements Client#close method, allows Infinity for "http2SessionIdleTime" parameter
+- Adds missed tests for FQL queries
+- Fixes cross-platform global object
+- Notifies about new package version and shows release notes on install
+- Exposes parseJSON from utils
+- Adds 'ngram' to special cases for snake_case to CamelCase conversion
+- Adds load tests
+- Automatically create jira ticket on github issue
+- Fixes undefined value in let return an empty object
```

Comment:
I'm having trouble parsing what this means. Is this a fix for the return value from calling `Let`?

Reply:
Yes, it looks like it is:
https://github.com/fauna/faunadb-js/pull/477
does this sound better? `Fixes return value from calling Let if a key is set to undefined`

Reply:
```suggestion
- Fixes handling of `Let` bindings when a key's value is undefined
```

Reply:
cool, thanks